Expensify/expensify-common
# Fix/14060: Bug: Link with multiline text loses markdown when edited

### Fixed Issues
$ https://github.com/Expensify/App/issues/14060

# Tests
1. Login with any account and go to any chat room
2. Send this message:
```
[multiline
text](https://google.com/)
```
3. Edit this message

# QA

1. Login with any account and go to any chat room
2. Send this message:
```
[multiline
text](https://google.com/)
```
3. Edit this message
4. Verify that when editing, the original message should be auto-filled

```
[multiline
text](https://google.com/)
```
